conf/defaults.yaml

```diff
@@ -259,6 +259,10 @@ topology.disruptor.batch.size: 100
 topology.disruptor.batch.timeout.millis: 1
 topology.disable.loadaware: false
 topology.state.checkpoint.interval.ms: 1000
+topology.metrics.aggregate.per.worker: false
```

Comment:
@HeartSaVioR Just to be clear, please don't take my comments to be a criticism of your work. You've done a fantastic job with metrics improvements.

Through that work it's become apparent that the existing API for metrics consumers is lacking in a number of ways. I view this as an opportunity to correct that.

We're incrementally moving toward semantic versioning. I'm not opposed to breaking backward compatibility with the existing metrics API and moving this branch to 2.x. The fact that master is "2.0" is just a technicality that can easily be changed.

I'd like to fix the API to better reflect the work you've done, rather than to implement a one-off fix.